docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/activation/identity-15.rst

```diff
@@ -0,0 +1,48 @@
+Identity
+========
+
+
+.. meta::
+  :description: Learn about Identity-15 - a simple operation that forwards the input to the output.
+
+**Versioned name**: *Identity-15*
+
+**Category**: *Matrix*
+
+**Short description**: The *Identity* operation forwards the input to the output.
+
+**Detailed description**: The *Identity* operation generates a new tensor that mirrors the input tensor in shape, data type, and content, effectively implementing the linear activation function f(x) = x.
+This operation creates a copy of the input data, therefore any modifications to the output tensor do not impact the original input tensor.
```

Comment:
I wonder is there a need to strictly specify that data is always a copy.
Maybe it can be "implementation defined" to allow plugins for some optimizations?

Reply:
Added the information about input_ptr == output_ptr optimization